v22.1/map-sql-activity-to-app.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Map SQL Activity using an Application Name
+summary: How to set an application name and filter SQL activity associated with the application.
+toc: true
+docs_area: develop
+---
+
+Many relational database management systems enable you to programmatically create and tag your SQL connections with a custom name or label. This practice lets you to quickly identify which part of your application is potentially degraded due to database health and design. By identifying problematic transactions and statements, you can quickly trace back to parts of your applications that generated those transactions and statements for a holistic understanding of the application performance impact as well as potential mitigations and optimization opportunities.
+
+## Set the application name
+
+It is best practice to set the application name with CockroachDB. You can do this in the [connection string](connection-parameters.html#additional-connection-parameters) `postgres://root@<servername>:26257/mydb?movr_app` or at the [session level](set-vars.html):
```

Comment:
We're missing the 'application_name parameter' in the connection string. 

`postgres://root@<servername>:26257/mydb?application_name=movr_app`